<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EditorTooltipsWidget</class>
 <widget class="QWidget" name="EditorTooltipsWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>637</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QCheckBox" name="chkShowFunctionTips">
     <property name="text">
      <string>Show function tips</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="grpEnableTooltips">
     <property name="title">
      <string>Enable mouse hover tooltips</string>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QCheckBox" name="chkIssueTooltips">
        <property name="text">
         <string>Show syntax issue tooltips</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="chkHeaderTooltips">
        <property name="text">
         <string>Show full header filename tooltips</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="chkIdentifierTooltips">
        <property name="text">
         <string>Show identifier definition tooltips</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="chkDebugTooltips">
        <property name="text">
         <string>Show expression value tooltips when debugging</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
